app/logic/signinKiosk.py

```diff
@@ -0,0 +1,21 @@
+
+from flask import Flask
+from app.models.eventParticipant import EventParticipant
+from app.models.user import User
+
+def sendkioskData(bnumber, eventid):
+    """Accepts scan input and signs in the user. If user exists or is already
+    signed in will return flasher"""
+    bNumberToUsername = User.select().where(User.bnumber == bnumber).objects()[0]
+    attendedParticipants = (EventParticipant.select(EventParticipant.user)

```

Comment:
Rather than do the query, create a list, and then check if the user is in the list, just check for the user directly in the query. If nothing comes back, they aren't in the list.